### PR TITLE
added new optional arguments for visibility, message retention and

### PR DESCRIPTION
**Pull request by tuxpower**
receive wait time

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -10,6 +10,21 @@ variable "max_message_size" {
   default = "262144"
 }
 
+variable "visibility_timeout_seconds" {
+  description = "The visibility timeout for the queue."
+  default = "43200"
```
Comment (antonbabenko) [inline variables.tf]: Remove quotes from integer

File: variables.tf [hunk 2]
```diff
@@ -10,6 +10,21 @@ variable "max_message_size" {
...
+}
+
+variable "message_retention_seconds" {
+  description = "The number of seconds Amazon SQS retains a message."
+  default = "1209600"
```
Comment (antonbabenko) [inline variables.tf]: same here

File: variables.tf [hunk 3]
```diff
@@ -10,6 +10,21 @@ variable "max_message_size" {
...
+}
+
+variable "receive_wait_time_seconds" {
+  description = "The time for which a ReceiveMessage call will wait for a message to arrive (long polling) before returning."
+  default = "0"
```
Comment (antonbabenko) [inline variables.tf]: and here :)

**Review by antonbabenko — COMMENTED** [review]
Minor comments, the rest LGTM

**Comment by tuxpower** [comment]
Done. @antonbabenko can you please review?

**Pull request merged**